images/pegasus/Dockerfile

```diff
@@ -55,103 +52,70 @@ RUN \
 		apt-get autoremove -y && \
 		rm -rf /var/lib/apt/lists/*
 
-# Downloading PCSX2 AppImage
-ARG PCS2_APP_IMAGE_URL=https://github.com/PCSX2/pcsx2/releases/download/v1.7.5232/pcsx2-v1.7.5232-linux-appimage-x64-Qt.AppImage
-RUN \
-    echo "**** Downloading PCSX2 AppImage ****" && \
-        mkdir -p /tmp && \
-        cd /tmp && \
-		wget -O pcsx2-emu-Qt.AppImage "${PCS2_APP_IMAGE_URL}"
+# Install pegasus
+RUN  <<_INSTALL_PEGASUS
+  #!/bin/bash
+  set -e
+
+  echo "**** Installing pegasus ****"
+  apt-get update -y
+  apt-get install -y --no-install-recommends jq curl
+
+  # Pegasus still requires libssl1.1, but ubuntu switched to libssl3. Need to manually install it.
+  # TODO: Doing this could be a terrible idea (it does work)
+  wget http://archive.ubuntu.com/ubuntu/pool/main/o/openssl/libssl1.1_1.1.1f-1ubuntu2_amd64.deb --output-document=libssl1.1.deb
+  apt-get install -y ./libssl1.1.deb
+
+  # Pegasus doesn't tag releases, they have a rolling weekly release
+  # so we have to parse the Github APIs to get the link
+  curl -s https://api.github.com/repos/mmatyas/pegasus-frontend/releases/tags/continuous | \
+  jq -r '.assets[] | select(.name | contains(".deb")) | .browser_download_url' | \
```

Comment:
OMG - You already did it!

Reply:
If you're getting the latest version from continuous, we "may" be able to skip the `libssl1.1.deb` bit. I was in their discord channel a while back and they mentioned that they had updated the code to use the newer `libssl3`.